stdlib/experimental/textreader/textreader.flux

```diff
@@ -0,0 +1,33 @@
+// Package array provides functions for building tables from Flux arrays.
+//
+// introduced: 0.103.0
+// tags: text,tables
+package textreader
+
+
+// from constructs a table from an arbitrary sample of text
```

Comment:
```suggestion
// from constructs a table from an arbitrary sample of text.
```

Is this "arbitrary"? From the example below, does the text need to be structured in a specific way? Columns defined by pipes? Could one use (or define) another separator?